### DISCUSSION TITLE
Question concerning Data storage on Synology

### DISCUSSION THREAD

**Discussion by LaKai-dot** [felddy/foundryvtt-docker#177] [Q&A] [answered]
Hello, my name is Kai from Germany and i do have almost no experience in coding or special software aso. I am a pure User, and every step i take, starting from setting up the Synology, up to use Docker and setting up Felddys Foundry Container was a lotz of work for me.

But in the end i succeeded (so far): Yiehah

But now i have two questions:

1. Where is the Foundry Data stored on my Synology?
- I am logged in with an administrator account
- I have created a folder path "/home/Foundry/"
- I have set the FOUNDRY_HOME to "/home/Foundry"
- I have not set a path for the volume
- The container is running, and i can reach Foundry on my Synology and also from extern
- But i cannot find folders for Foundry Data on my Synology. The folders mentioned before seem to be empty

2. How do i add Modules stored on my Synology to my Foundry installation? How do i set the path to the Module within Foundry?

Thanks a lot for your help,
Kai

**Comment by LaKai-dot** [comment]
Would it normally be possible to see the data stored under "/data"?

**Comment by LaKai-dot** [comment, marked as answer]
Solved: It seems to be importnat to create the folder path under "Volume" via the volume folder creation, and to not create an folder "data" on your own.

**Reply by felddy** [reply]
I'm glad you got it solved.  Sorry I wasn't able to reply earlier.  

One thing to note:  You shouldn't be setting the `FOUNDRY_HOME` environment variable.  That is used internally by the container and is not meant to be changed.   If I recall correctly the Synology Docker GUI has a bad habit about exposing variables internal to the container.  Also be careful that it doesn't pin `FOUNDRY_VERSION` on you. 

You specify where to store the data on your NAS by mapping the location to the container's volume: https://github.com/felddy/foundryvtt-docker#volumes
